### PR TITLE
[DST-64]: Ref: unify the used chev icon over all components

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

File: packages/components/src/Accordion/AccordionItem.tsx [hunk 1]
```diff
@@ -9,6 +9,27 @@ import { SVG, useClassNames, useStateProps } from '@marigold/system';
...
 import { Button } from '../Button';
 
+// Select Icon
+// ---------------
+interface ChevronProps {
+  className?: string;
+}
+// icons
+// -----------------
+const ChevronDown = ({ className }: ChevronProps) => (
```
Comment (sebald) [inline packages/components/src/Accordion/AccordionItem.tsx:19]: Move to own component and reuse in all components? 🤔

Comment (OsamaAbdellateef) [reply packages/components/src/Accordion/AccordionItem.tsx]: @sebald 
I did some changes now

**Comment by sarahgm** [comment]
@OsamaAbdellateef  conflicts

**Comment by OsamaAbdellateef** [comment]
> @OsamaAbdellateef conflicts

@sarahgm 
Resolved

**Review by sarahgm — APPROVED** [review]

**Pull request merged**